docs/cli/options.md

```diff
@@ -0,0 +1,3 @@
+# Options
```

Comment:
Not sure this warrants its own page if we are going to link out we could include it in the top-level page?